peerdroid/src/main/java/rdx/works/peerdroid/data/PeerdroidLink.kt

```diff
@@ -33,9 +43,36 @@ interface PeerdroidLink {
      * Call this function to add a connection in the wallet settings.
      *
      */
```

Comment:
maybe it would be nice to update the comment here with a short explanation what this class does now.